src/transaction/flashback.c

```diff
@@ -64,9 +64,88 @@ static volatile int flashback_Threshold_to_remove_archive = 0;	/* If the differe
 								 * the archive log volume can be deleted.
 								 */
 // *INDENT-OFF*
-static std::atomic_bool flashback_Is_active = false;	// the status value that the flashback is processing the request
+static std::atomic_bool flashback_Is_active = false;	// the status value that the flashback is processing the a request
+static std::atomic_bool flashback_In_progress = false;  // the status value that the flashback is in progress
 // *INDENT-ON*
 
+static CSS_CONN_ENTRY *flashback_Current_conn = NULL;	// the connection entry for a flashback requests
+
+/*
+ * flashback_is_duplicated_request - check if the caller is duplicated request for flashback
+ *
+ * return   : duplicated or not
+ */
+
+bool
+flashback_is_duplicated_request (THREAD_ENTRY * thread_p)
```

Comment:
How about the following code flow? IMO, it is more readable.

```
if (flashback_In_progress == false)
{
    // 이전 flashback 연결이 정상적으로 종료된 경우. 현재 수행 중인 flashback 없음.
    return false;
}
else
{
    if (flashback_Current_conn->in_flashback == true)
    {
        // 이전 flashback 요청이 아직 처리 중인 경우
        return true;
    }
    else
    {
        // 이전 flashback 요청이 비정상 종료된 경우
        return false;
    }
}
```

Reply:
Does this code mean to remove `if(previous_conn->status == CONN_OPEN)`? 
or 
Does it mean to first classify whether or not flashback is normally terminated using `flashback_In_progress`?

Reply:
@joohok Both of them. I think it is good to discuss off-line first.

Reply:
OK, lets discuss it off-line.